### ISSUE TITLE
Source text for all sources?

### ISSUE THREAD

**Issue by offendo** [nelson-liu/evaluating-verifiability-in-generative-search-engines#1]
Hi there, is it possible for you to release the source text for all of the sources in your human annotated data? There are several links which appear in the human annotated examples but not in the verifiability judgements dataset, so I can't find any source text. 

Ideally, I think a separate document containing only sources you've used would be very nice, e.g.:

```python
{
  “source_name": ...,
  “source_url": ...,
  # ... other metadata
  “source_text": ...,
  “source_raw_text": ...,
}
```

Alternatively, the code & running instructions you used to scrape the text. I saw you have a short tutorial on the readme but I couldn't figure out the chrome DOM distiller. 

Much appreciated!

**Comment by nelson-liu** [comment]
Hi!

> Alternatively, the code & running instructions you used to scrape the text. I saw you have a short tutorial on the readme but I couldn't figure out the chrome DOM distiller.

The output of the chrome DOM distiller is the contents of the verifitability judgements dataset . There are some sites that I wasn't able to scrape (due to, e.g., cloudflare or just generally paywalls), hence why there are some source_urls in the human evaluation judgments that don't appear in the verifiability judgments,

When we annotated the data, we didn't pre-scrape these sites up-front---the verifiability judgments dataset was created post-hoc and is not actually used in the paper, we just figured it'd be useful for future work. When collecting the human evaluation judgments, we had annotators manually visit every source url (so they also get the full experience, since scraping often makes it hard to preserve dynamic page elements like JS).

Hope that answers your question! TL;DR is that I don't have the source text for all source_urls either.
